### ISSUE TITLE
Winget Works

### ISSUE THREAD

**Issue by helmanfrow** [ytdl-org/youtube-dl#31520]
## Checklist

- [ ] I'm asking a question
- [ ] I've looked through the README and FAQ for similar questions
- [ ] I've searched the bugtracker for similar questions including closed ones

## Question

WRITE QUESTION HERE
Winget is not mentioned as an installation method but it appears to work nonetheless. Is this not recommended?
![image](https://user-images.githubusercontent.com/38217564/217394899-6bc684de-8277-4006-8796-6ff4e201bc52.png)

**Comment by dirkf** [comment]
Thanks for letting us know.

Apparently  Microsoft invented _winget_ since the documentation was last updated and a version of _youtube-dl_ [was added to its package repository](https://github.com/microsoft/winget-pkgs/pull/68800) last August. The submitter did not coordinate with the project and we have no way of guaranteeing that the program listed there actually is or will be _youtube-dl_. Having said that, it does appear to be so currently. The project has no process for automatically updating or submitting a new version.

It would be useful to hear from Windows users if _winget_ submission should be added to the formal release process.

**Comment by pukkandan** [comment]
imo it should just be treated like all the other third party packages (choco, apt etc)

**Comment by helmanfrow** [comment]
Yes, why not add it to Wiinget? It's convenient and built into Windows. I use it often.
